tutorials/Homology-Modelling/homology-modelling-tutorial.ipynb

```diff
@@ -27,22 +29,36 @@
     "\n",
     "- Are there any other reasons why you might not want to select a particular structure? I.e. inaccurate assignemnt of EM density - missing loops or regions. \n",
     "\n",
-    "We will use this jupyter notebook to explore this.  The idea of this tutorial is that you should be able to follow it through at your own pace.  If you are of a timetabled course, then there will be some demonstrator help available, but this tutorial has been designed such that you should be able to complete it without help, even if you get stuck.\n",
+    "We will use this jupyter notebook to explore this.  If you are following the OxCompBio course, then an introduction to python and juypter notebooks is the first session.  If not then you can either work your way through that (found elsewhere in this repo) or see this quick (7 minute) https://www.youtube.com/watch?v=jZ952vChhuI introduction.\n",
```

Comment:
Personally I think it's best we don't start sending the students to different places for external resources. Can we just tell them to look at notebook 1 in the python course and leave it as that?

Reply:
fair enough

Reply:
I'll create an issue to reflect the need to have a jupyter external resource like this one though.